logging/scheduling_resources/logging-taints-tolerations.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="logging-taints-tolerations"]
+= Using taints and tolerations to control logging pod placement
+:context: logging-taints-tolerations
+
+toc::[]
+
+Taints and tolerations allow the node to control which pods should (or should not) be scheduled on them.
```

Comment:
... can or can not ... ?

I'm not certain if there is a distinct difference between the choice of words but "can", to me, sounds more gating thought I'm not hard pressed to these terms.


There is another bit related to taints and tolerations that is not exactly conveyed by this statement.  They also are a way of conveying which pods get booted first.  For example, a container could be allowed on a node.  As more containers are scheduled there, they require more CPU and memory.  As you get to capacity, the scheduler will boot containers from the node.  This is where it will consider the "tolerations".  If you have stated your pod can tolerate "memory" pressure, it may boot some other workload so that yours can continue.

Reply:
I am reusing this line from https://docs.openshift.com/container-platform/4.14/nodes/scheduling/nodes-scheduler-taints-tolerations.html just because we needed something as an abstract for this section. Since this PR is mostly about restructuring the content, let's follow up any changes or additions to the abstracts in an additional PR to avoid scope creep.